src/utils/solana/splTokenRegistry.ts

```diff
@@ -1,5 +1,35 @@
-import { TokenInfo, TokenListProvider } from '@solana/spl-token-registry'
+import axios from 'axios'
 
+const TOKEN_REGISTRY_URL =
+  'https://cdn.jsdelivr.net/gh/solana-labs/token-list@main/src/tokens/solana.tokenlist.json'
+
+const TOKEN_REGISTRY_URL_FALLBACK =
+  'https://raw.githubusercontent.com/solana-labs/token-list/main/src/tokens/solana.tokenlist.json'
+
+type TokenInfo = {
+  chainId: 101 | 102 | 103
+  address: string
+  symbol: string
+  name: string
+  decimals: number
+  logoURI: string
+  tags: string[]
+}
+
+async function fetchTokens() {
+  const cdnRegistryFetch = axios.get(TOKEN_REGISTRY_URL)
+  const fallbackRegistryFetch = axios.get(TOKEN_REGISTRY_URL_FALLBACK)
```

Comment:
Why load both here if the primary is almost always going to succeed? The user's bandwidth is going to be wasted for no reason.